packages/orm/src/index.ts

```diff
@@ -107,7 +112,21 @@ interface Flavoring<FlavorT> {
 export type Flavor<T, FlavorT> = T & Flavoring<FlavorT>;
 
 export function setField(entity: Entity, fieldName: string, newValue: any): void {
-  ensureNotDeleted(entity);
+  ensureNotDeleted(entity, { ignore: "pending" });
+  const em = getEm(entity);
+
+  if (em.isFlushing) {
+    const { context } = em["contexty"];
```

Comment:
This is fine but I think the idea with contexty is that this could also be `const { context } = the-module-const`.